melvs/melvsboot/src/main/java/com/melvs/melvsboot/BusinessService.java

```diff
@@ -0,0 +1,21 @@
+package com.melvs.melvsboot;
+
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+import java.util.List;
+
+//service once spring boot started will load to memory
+@Service
```

Comment:
Interesting usage here of @Service. For me I used @Beans and @Autowired annotations to initialize my test data to memory.

Is @Service connected to any other code in your project? Like how @Beans and @Autowired connect to each other

Reply:
added the tutorial link on Business service in our shared docs.

Reply:
We would be using annotations Service, Component, Entity, etc. under spring. There is a set of annotations that I am making standard and others for special use cases. We would be using Autowired for special cases and beans for classes used under Configuration annotation.